src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -21632,6 +21632,20 @@ bool Unit::IsInDisallowedMountForm() const
     return false;
 }
 
+void Unit::SetUInt32Value(uint16 index, uint32 value)
+{
+    Object::SetUInt32Value(index, value);
+
+    switch (index)
```

Comment:
Hm, why a switch if there's only one case?

Reply:
Just wanted to draw attention to the possibility of similar cases for other fields. If there are more, you can simply extend the switch.